server/src/controllers/AuthController.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
       visit https://stackoverflow.com/questions/3825990/http-response-code-for-post-when-resource-already-exists for more details
       */
       if (user1) {
-        return res.status(422).json({
+        res.status(422).json({
```

Comment:
Personally I prefer the return. Altho the value is useless, it prevents accidental extra calls to res.json etc